### PR TITLE
chore: Add `@pagopa/react-native-nodelibs` support

### PR DESCRIPTION
**Pull request by grausof**
### Short description
This PR introduce [react-native-nodelibs](https://github.com/pagopa/react-native-nodelibs) to use node modules without nodeify. 

[react-native-nodelibs](https://github.com/pagopa/react-native-nodelibs) is a fork of [node-libs-react-native](https://github.com/parshap/node-libs-react-native) which introduces new libraries and support for the latest versions of react native.

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -10,9 +10,11 @@
     "build:ios": "cd ios && xcodebuild -workspace IoReactNativeProximityExample.xcworkspace -scheme IoReactNativeProximityExample -configuration Debug -sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO"
   },
   "dependencies": {
+    "@pagopa/io-react-native-nodelibs": "^0.0.2",
     "react": "18.2.0",
     "react-native": "0.72.6",
     "react-native-ble-manager": "^10.1.5",
+    "react-native-randombytes": "^3.6.1",
```
Comment (hevelius) [inline example/package.json:17]: `react-native-random-bytes` is marked as deprecated. What do you think to use `https://www.npmjs.com/package/react-native-get-random-values` as already used in `io-app`?

**Review by hevelius — CHANGES_REQUESTED** [review]
I have tested in a RN app and works as expected. The only thing concerns the installation, in fact the `react-native-nodelibs` package does not seem to exist on npm while `io-react-native-nodelibs` does.

**Comment by grausof** [comment]
> I have tested in a RN app and works as expected. The only thing concerns the installation, in fact the `react-native-nodelibs` package does not seem to exist on npm while `io-react-native-nodelibs` does.

The package had been renamed but not yet deployed. Now everything should be correct. https://www.npmjs.com/package/@pagopa/react-native-nodelibs

File: example/package.json (hunk 1 above)
Comment (grausof) [reply example/package.json]: I removed the dependency by introducing the new version of `react-native-nodelibs`

**Review by hevelius — APPROVED** [review]
LGTM

**Pull request merged**